konfera/models/order.py

```diff
@@ -53,3 +53,10 @@ def to_pay(self):
     @property
     def variable_symbol(self):
         return str(int(self.uuid))[:10]
+
+    @property
+    def event(self):
+        if len(self.ticket_set.all()) > 0:
+            return self.ticket_set.all()[0].type.event
+        else:
```

Comment:
remove the `else` and `return None` directly in `event` ( = same behavior nicer code)